src/develop/pixelpipe_hb.c

```diff
@@ -890,7 +890,7 @@ static void _pixelpipe_pick_live_samples(const float *const input, const dt_iop_
 
   // display rgb --> histogram rgb
   if(display_profile && histogram_profile)
-    xform_rgb2rgb = cmsCreateTransform(display_profile, TYPE_RGB_FLT, histogram_profile, TYPE_RGB_FLT, INTENT_PERCEPTUAL, 0);
+    xform_rgb2rgb = cmsCreateTransform(display_profile, TYPE_RGB_FLT, histogram_profile, TYPE_RGB_FLT, INTENT_RELATIVE_COLORIMETRIC, 0);
```

Comment:
Likewise, `DT_INTENT_RELATIVE_COLORIMETRIC`

Reply:
Second thoughts -- as this is a call into LCMS, should use the LCMS constant.

Reply:
No change needed here then :)